frontend/packages/console-dynamic-plugin-sdk/src/validation/ExtensionValidator.ts

```diff
@@ -29,13 +75,24 @@ export const collectCodeRefData = (extensions: Extension[]) =>
 export const findWebpackModules = (
   compilation: webpack.Compilation,
   exposedModules: ConsolePluginBuildMetadata['exposedModules'],
+  basePath?: string,
 ) => {
   const webpackModules = Array.from(compilation.modules);
-
   return Object.keys(exposedModules).reduce((acc, moduleName) => {
+    const absolutePath =
+      basePath && guessModuleFilePath(path.resolve(basePath, exposedModules[moduleName]));
+
     acc[moduleName] = webpackModules.find((m) => {
-      const rawRequest = _.get(m, 'rawRequest') || _.get(m, 'rootModule.rawRequest');
-      return exposedModules[moduleName] === rawRequest;
+      // first strategy: check if the module name matches the rawRequest
+      const rawRequest: string = _.get(m, 'rawRequest') || _.get(m, 'rootModule.rawRequest');
```

Comment:
Any reason why we're using `_.get` instead of direct object property access?

We should generally try to avoid using Lodash helpers as much as possible.

Reply:
`_.get` was what was already used. I'll change it to a direct access